docs/best-practices.md

```diff
@@ -0,0 +1,51 @@
+# Best practices for writing actors
+
+
+### Follow the Python Coding Guidelines
+
+See the [Python Coding Guidelines](CONTRIBUTING.md#coding-guidelines).
+
+### Avoid running the code on a module level
+
+Actors are completely written in Python, however, they are loaded beforehand to get all the meta-information from the actors.
+To avoid slow downs we ask actor writers not to execute any code that is not absolutely necessary on a module level.
+
+### Avoid global non Leapp imports
+
+Avoid importing system or bundled libraries on a module level. This again has to do with slow down, and also to avoid
+unnecessary complications for our tests automation which needs to inspect the actors beforehand.
+
+### Use the snactor tool
```

Comment:
Maybe some link for the first tutorials we have under Read The Docs, talking about how to use snactor tool?

Reply:
Added.